docs/docs-content/clusters/data-center/nutanix/register-nutanix-cloud.md

```diff
@@ -0,0 +1,263 @@
+---
+sidebar_label: "Register Nutanix Cloud"
+title: "Register Nutanix Cloud"
+description: "Learn how to register a Nutanix cloud in Palette."
+hide_table_of_contents: false
+sidebar_position: 5
+tags: ["data center", "nutanix"]
+---
+
+
+A system administrator registers the Nutanix cloud in Palette by invoking system-level APIs. These APIs provide specific cloud information, the cloud logo, and the key-value pairs required to add the cloud to Palette. They also enable uploading the YAML templates used to create the cluster, control plane, and worker nodes. This section provides instructions on how to download and modify the YAML templates, as well as how to use the APIs to register a Nutanix cloud to Palette.
+
+
+## Prerequisites
+
+
+- A Nutanix cloud created with the minimum supported CAPI version 1.5.3. Refer to the Nutanix [Validated Integrations](https://opendocs.nutanix.com/capx/v1.2.x/validated_integrations/#validated-versions) compatibility matrix.
+
+- The Nutanix CAPI version must be compatibe with Palette's CAPI version.
+
+- A Cluster API (CAPI) image created for the Nutanix cloud platform. For guidance, refer to [Building CAPI Images for Nutanix Cloud Platform](https://image-builder.sigs.k8s.io/capi/providers/nutanix). Images can be found in the Nutanix Prism dashboard under **Compute & Storage**.
+
+- A Palette account with system console access. The user with this privilege is the *admin user* of the self-hosted [Palette](https://docs.spectrocloud.com/enterprise-version/system-management/#system-console) or [VerteX](https://docs.spectrocloud.com/vertex/system-management/#system-console) instance.
+
+- A Nutanix logo downloaded. Review logo requirements in [Register the Cloud](#register-the-cloud).
+
+- [`curl`](https://curl.se/docs/install.html) command installed.
+
+<!-- - A valid Palette authentication token. To learn how to acquire an authentication token, review the [Authorization Token](https://docs.spectrocloud.com/user-management/authentication/authorization-token) guide. -->
+
+
+## Setup
+
+Use the following steps to prepare to register your cloud with Palette. 
+
+### Customize YAML Configuration Files
+
+1.  Download the following YAML files from the [Nutanix Cluster API (CAPI) Provider](https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix) GitHub repository:
```

Comment:
Those should actually be downloaded from a specific version of CAPX, in the [release page](https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/releases). For example for 1.2.4, you scroll down to the [assets](https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/releases/tag/v1.2.4). This is important!

<img width="1234" alt="Screenshot 2023-12-21 at 18 15 33" src="https://github.com/spectrocloud/librarium/assets/10382023/4a7eb5e8-936e-4bec-ad71-763e1a82d17b">

Reply:
Removed curl commands and revised step 1 to say:

1.  Download the following YAML files from a specific version of CAPX found on the [Nutanix Releases](https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/releases) page in the GitHub repository.

Reply:
I think it is a bit confusing now. Which specific version of CAPX? I liked the curl commands better.
What if we create an env variable to store the CAPx version and keep the curl commands?